gulpfile.js

```diff
@@ -514,9 +529,10 @@ gulp.task('get-release-checksums', function(done) {
     const releasePath = `./dist_${type}/release`;
 
     let files = fs.readdirSync(releasePath);
-
+    
+    shell.cd(releasePath);
```

Comment:
Better to pass `{ cwd: ... }` option to `shell.exec` below, see https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback